TUTORIAL.md

```diff
@@ -0,0 +1,110 @@
+# Badges and Schedules
+
+## Objectives
+
+1. Utilize iteration
+2. Define methods that use iteration and control the return values of those methods
+3. Define methods that call other methods
+
```

Comment:
@danny-sullivan Can you please remove the objectives from the TUTORIAL.md. The README already has the Objectives listed out.